traffic_ops/app/conf/production/riak.conf

```diff
@@ -1,5 +1,7 @@
-
 {
 	"user": "riakuser",
-	"password": "password"
+	"password": "password",
+	"tlsConfig": {
+		"insecureSkipVerify": true
```

Comment:
Do we need this in prod?

Reply:
Didn't test in prod and I'm not sure if it would work in prod without it as you would have to set the ServerName in TLSConfig and I think ServerName verification would fail when selecting a RIAK node from a cluster of nodes.